test/acceptance/docker-compose.yaml

```diff
@@ -7,24 +7,16 @@ services:
     depends_on:
       - selenium
       - hub
-      - browserstack
-
     environment:
       TEST_ENVIRONMENT_ROOT_URL: ${TEST_ENVIRONMENT_ROOT_URL}
       CHROME_ARGS: "--headless --ignore-certificate-errors"
-          
+      BROWSERSTACK_USERNAME: ${BROWSERSTACK_USERNAME}
+      BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY}
+
   selenium:
     image: selenium/standalone-chrome-debug
     container_name: chrome-browser
 
   hub:
     image: selenium/hub
     container_name: selenium-hub
-
-  browserstack:
-    image: browserstack/local
-    container_name: browserstack-local
-    environment:
-      BROWSERSTACK_USERNAME: ${BROWSERSTACK_USERNAME}
-      BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY} 
-      
```

Comment:
Don't need this image as we use the npm package as the tunnel to browserstack web